### PR TITLE
[#5261] feat(oceanbase-catalog): Add IT for OceanBase JDBC catalog

### PR DESCRIPTION
**Pull request by yuanoOo**
### What changes were proposed in this pull request?

Add IT for OceanBase JDBC catalog.

### Why are the changes needed?

Fix: #5261

### Does this PR introduce _any_ user-facing change?

No

### CODE REVIEW

File: catalogs/catalog-jdbc-oceanbase/src/test/java/org/apache/gravitino/catalog/oceanbase/integration/test/CatalogOceanBaseDriverIT.java
Comment (mchades) [inline catalogs/catalog-jdbc-oceanbase/src/test/java/org/apache/gravitino/catalog/oceanbase/integration/test/CatalogOceanBaseDriverIT.java:1]: What is this class for?

Comment (yuanoOo) [reply catalogs/catalog-jdbc-oceanbase/src/test/java/org/apache/gravitino/catalog/oceanbase/integration/test/CatalogOceanBaseDriverIT.java:1]: Removed it.

File: catalogs/catalog-jdbc-oceanbase/src/test/java/org/apache/gravitino/catalog/oceanbase/integration/test/CatalogOceanBaseIT.java
Comment (mchades) [inline catalogs/catalog-jdbc-oceanbase/src/test/java/org/apache/gravitino/catalog/oceanbase/integration/test/CatalogOceanBaseIT.java:1]: I would suggest the name: `CatalogOceanBaseIT`

**Review by mchades — APPROVED** [review]
LGTM. Thanks for your contributions!

**Pull request merged**